BackToBasicsUIKit/BackToBasicsUIKit.xcodeproj/project.pbxproj

```diff
@@ -453,7 +501,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = EER35MGDHD;
+				DEVELOPMENT_TEAM = 9GX9P2M56N;
```

Comment:
Please discard both of these buildSettings changes. You can change them to test your code in your device but don't forget to not commit them.